server/routes/createGoal/createGoalController.test.ts

```diff
@@ -353,6 +353,10 @@ describe('createGoalController', () => {
         },
       ] as Array<NewGoal>
 
+      req.body = {
```

Comment:
Existing unit tests for the `submitReviewGoal` controller method now need the `action: 'submit-form'` so that the contoller method follows the correct path / behaviour